portfolio/src/main/webapp/index.html

```diff
@@ -10,23 +10,47 @@
     <div id="content">
       <h1>Natalie Volk's Portfolio</h1>
       <p>Welcome to my portfolio! </p>
-      <p>I'm a second-year Engineering Science student at the University of Toronto, 
-          with a major in Machine Intelligence. I'm interested in 
-          software engineering, consulting, and machine learning. 
-          I'm currently interning as a SWE & Consulting intern at Deloitte. 
-          Outside of school and work, I love figure skating, reading, and going for long walks.
-      </p>
+      <div id="intro-section">
+        <div id="profile-photo">
+            <img src="/images/headshot.jpeg" />
+        </div>
+        <p>I'm a second-year Engineering Science student at the University of Toronto, 
+            with a major in Machine Intelligence. I'm interested in 
+            software engineering, consulting, and machine learning. 
+            I'm currently interning as a SWE & Consulting intern at Deloitte. 
+            Outside of school and work, I love figure skating, reading, and going for long walks.
+        </p>
+      </div>
       <p>Click <a href="/hello">here</a> to see content created by server code.</p>
-      <p>Click here to find a random fun fact about myself:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
-      <div id="images">
-          <img src="/images/headshot.jpeg" />
-          <p> I live in Toronto, Canada. Here's one of my favourite pictures that I've taken 
-              of the Toronto skyline:
-          </p>
+      <div id="fun-fact">
+        <p>Click here to find a random fun fact about myself:</p>
+        <button onclick="addRandomGreeting()">Fun Facts</button>
```

Comment:
Since this is no more a greeting but a fun fact. For code readability it would be better if the method reflects that. Try renaming it to something more apt.